lib/diagnostic.rb

```diff
@@ -89,22 +94,22 @@ def method_name
 
 # Predict what value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'Just your average billionaire'
 
```

Comment:
the correct answer is Diagnostic.flow_control = 'The Dark Knight'

what is truthy and what is falsy? what euvals first?